tests/end2end/utils/operations.py

```diff
@@ -33,6 +33,7 @@ class OperationResult(Enum):
     SUCCESS = "Success"
     EXCEPTION = "Exception"
     FORBIDDEN = "Forbidden"
+    NOT_AVAILABLE = "Not Available"
```

Comment:
Which is the difference of NOT_AVAILABLE and the others? Can be added on the class comment above?

Reply:
Yup. This was for an operation that was allowed to be made but whose expected result is a not-answer, a timeout. The server wont be giving errors, because the public API is available, but the device we will send the jobs to will not be processing them, and they will be stuck in `PENDING` state as an intended behaviour that we want to check for. I'll add it to the docstrings

Reply:
In more generic terms, the idea was, a situation where the communication with the API is successfull, but the resource you are asking for is not available at the moment, turning into an intended response that is not an error but that also implies that what you were requesting will not be done _at the moment_.
In specific terms and with the concrete examples, one is asking for a device and finding it is not `ONLINE`, and other is issuing executions that do not evolve into `QUEUED`, `COMLETED` or `EXCEPTION` for an `OFFLINE` device.